onebusaway-android/build.gradle

```diff
@@ -148,4 +151,8 @@ dependencies {
     compile 'org.onebusaway.util:comparators:1.0.0'
     // Only used on Amazon devices via amazon build flavor
     amazonCompile('com.amazon.android:amazon-maps-api:2.0')
+
+    //Apache http connection for images
```

Comment:
@cagryInside Can you use the native Android HttpURLConnection for this instead?  If not, we should include comments here or in code why.  See http://android-developers.blogspot.com/2011/09/androids-http-clients.html for why the HttpURLConnection is preferred to Apache libaries.